src/tab.group.ts

```diff
@@ -119,7 +119,7 @@ export default class TabGroup extends LitElement {
               })}
               data-test="overflow-end-button"
               tabindex="-1"
-              @click=${this.#onOverflowEndButtonClick}
```

Comment:
`#onOverflowEndButtonClick()` was just an indirection that called `#scrollTabsList()`. So I removed it as a middle-man and renamed `#scrollTabsList()` to `#onOverflowEndButtonClick()`.